debian/control

```diff
@@ -15,21 +15,38 @@ Standards-Version: 4.5.1
 
 Package: pimodbus-master
 Architecture: any
+Section: oldlibs
+Depends:
+  revpi-modbus-client,
+  ${misc:Depends}
+Description: transitional package
+  This is a transitional package. It can safely be removed.
+
+Package: pimodbus-slave
+Architecture: any
+Depends:
```

Comment:
```suggestion
Architecture: any
Section: oldlibs
Depends:
```

Reply:
I already fixed this earlier. Sorry for the inconvenience.